src/clj/game/core/shuffling.clj

```diff
@@ -21,6 +23,30 @@
     (swap! state update-in [:stats side :shuffle-count] (fnil + 0) 1)
     (swap! state update-in [side kw] shuffle)))
 
+(defn shuffle-cards-into-deck!
+  "Shuffles a given set of cards into the deck. Will print out what's happened. Will always shuffle."
+  [state from-side card shuffle-side & targets]
```

Comment:
Can this take `targets`, not `& targets`? Just force everyone to put things in a list or vector?